CMakeLists.txt

```diff
@@ -18,9 +18,9 @@ find_package(YCM REQUIRED)
 include(YCMDefaultDirs)
 ycm_default_dirs(${VARS_PREFIX})
 
+option(BUILD_SHARED_LIBS "Compile the library as a shared library" TRUE)
```

Comment:
This was just a renaming from https://github.com/robotology/yarp-wholebodyinterface/pull/59/files#diff-af3b638bc2a3e6c650974192a53c7291L22 to follow CMake standard practices , however I will add it.